torch/distributed/rpc/__init__.py

```diff
@@ -26,9 +27,8 @@ def init_rpc(
         backend=backend_registry.BackendType.PROCESS_GROUP,
```

Comment:
General comment, should `self_name` and `self_rank` be `name` and `rank`? For example, `init_process_group` uses `rank` and not `self_rank`.

Reply:
@mrshenli What do you think? If we all agree. I will do the change.

Reply:
+1 for this change (drop `self_` in the name)

Reply:
Updated.